src/Security/Authentication/JwtBearer/src/JwtBearerConfigureOptions.cs

```diff
@@ -39,26 +40,56 @@ public void Configure(string? name, JwtBearerOptions options)
             return;
         }
 
-        var issuer = configSection["ClaimsIssuer"];
-        var audiences = configSection.GetSection("Audiences").GetChildren().Select(aud => aud.Value).ToArray();
+        var issuer = configSection[nameof(TokenValidationParameters.ValidIssuer)];
+        var issuers = configSection.GetSection(nameof(TokenValidationParameters.ValidIssuers)).GetChildren().Select(iss => iss.Value).ToList();
+        if (issuer is not null)
+        {
+            issuers.Add(issuer);
+        }
+        var audience = configSection[nameof(TokenValidationParameters.ValidAudience)];
+        var audiences = configSection.GetSection(nameof(TokenValidationParameters.ValidAudiences)).GetChildren().Select(aud => aud.Value).ToList();
+        if (audience is not null)
+        {
+            audiences.Add(audience);
+        }
+
+        options.Authority = configSection[nameof(options.Authority)];
+        options.BackchannelTimeout = StringHelpers.ParseValueOrDefault(configSection[nameof(options.BackchannelTimeout)], _invariantTimeSpanParse, options.BackchannelTimeout);
+        options.Challenge = configSection[nameof(options.Challenge)] ?? options.Challenge;
+        options.ForwardAuthenticate = configSection[nameof(options.ForwardAuthenticate)];
+        options.ForwardChallenge = configSection[nameof(options.ForwardChallenge)];
+        options.ForwardDefault = configSection[nameof(options.ForwardDefault)];
+        options.ForwardForbid = configSection[nameof(options.ForwardForbid)];
+        options.ForwardSignIn = configSection[nameof(options.ForwardSignIn)];
+        options.ForwardSignOut = configSection[nameof(options.ForwardSignOut)];
```

Comment:
I'm not sure why some properties have ?? fallbacks and others don't. I'd expect them all to.